components/calendar/src/error.rs

```diff
@@ -64,259 +64,275 @@ pub enum DateError {
 }
 
 impl core::error::Error for DateError {}
```

Comment:
Thought: I don't love how this makes such a big diff. I like how in other crates we have the whole module structure in the lib.rs.